modules/getting-started/con-audit-log-file-rotation-overview.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+// assembly-audit-log.adoc
+//
+
+[id="con-audit-log-file-rotation-overview_{context}"]
+= About audit log file rotation
+
+Logging to a rotating file in {product} is helpful for persistent storage of audit logs.
+
+== Default settings
+
+Audit logging to a rotating file is disabled by default. When it is enabled, the default behavior changes to:
+
+* Rotate logs at midnight (local system timezone).
+* Log file format: `redhat-developer-hub-audit-%DATE%.log`.
+* Log files are stored in `/var/log/redhat-developer-hub/audit`.
+* No automatic log file deletion.
+* No gzip compression of archived logs.
+* No file size limit.
+
+Audit logs are written in the `/var/log/redhat-developer-hub/audit` directory.
+
+== Log file names
+
+Audit log file names are in the following format: `redhat-developer-hub-audit-%DATE%.log` where `%DATE%` is the format specified in [`auditLog.rotateFile.dateFormat`](#configuring-the-file-rotation-frequency). You can customize file names when you configure file rotation.
+
+== File rotation date and frequency
+
+Supported `auditLog.rotateFile.frequency` options include:
+
+* `daily`: Rotate daily at 00:00 local time
+* `Xm`: Rotate every `X` minutes (where X is a number between 0 and 59)
+* `Xh`: Rotate every `X` hours (where X is a number between 0 and 23)
+* `test`: Rotate every 1 minute
+* `custom`: Use `dateFormat` to set the rotation frequency (default if frequency is not specified)
+
+If `frequency` is set to `Xh`, `Xm` or `test`, the `dateFormat` setting must be configured in a format that includes the specified time component. Otherwise, the rotation will not work as expected.
```

Comment:
```suggestion
If `frequency` is set to `Xh`, `Xm` or `test`, the `dateFormat` setting must be configured in a format that includes the specified time component. Otherwise, the rotation might not function as expected.
```